bin/znapzend

```diff
@@ -164,6 +165,23 @@ just look at this dataset's backup plan, but iterate into its children that
 might have any. Useful for quick backups of a pool whose root dataset has no
 I<znapzendzetup> configurations defined, but some trees under it do.
 
+See also B<--inherited> for interaction with that option.
+
+=item B<--inherited>
+
+when backing up a particular dataset with B<--runonce>=[I<dataset>], do not
+require that the provided dataset directly has a locally defined backup plan
+configuration (sanity check that it is not "received" via ZFS replication).
+With this option in place, allow also to quickly snapshot and/or replicate
+datasets of subtrees which inherit a backup plan from a dataset which has it
+in its attributes whose source is "local".
+
+At this time, the B<--inherited> is effectively exclusive vs. B<--recursive>
+mode (and preferred if both are specified) if the requested dataset has some
+(local or inherited) backup plan. If a dataset without a backup plan is passed,
+the script should ignore the B<--inherited> option and just recurse until it
+finds some dataset(s) with a proper locally defined backup plan.
+
```

Comment:
adding an example use would be great to make the intended usage clear

Reply:
By the way, is there a way in this test suite to check what the calls actually did, not qust that they succeeded or failed with specific exit code? (Maybe grep stdout+stderr)

Here I can set up a few scenarios where I expect that e.g. these 1-2-3 configs are found (amount and names), but do not really know how to check for it programmatically.

Reply:
The documentation comments (for examples) are hopefully addressed by "Revise doc...." commit now, thanks.

Reply:
re testing ... the test suit is in perl and in shell, so yes anythihng is possible :) if you have a concrete example I may be able to advise